### DISCUSSION TITLE
Windows exe client 2024.1.0 cannot sync with self host server

### DISCUSSION THREAD

**Discussion by Vortilion** [dani-garcia/vaultwarden#4323] [Q&A]
The **Windows** client version **2024.1.0** seems to be unable to sync with my selfhosted vaultwarden server. If I login it shows an empty vault, and if I select File | Sync vault (using german client so not sure if thats the correct wording in the english client) it shows an error that "Sync has failed". The Browser-Plugin and the mobile app work perfectly.

![Screenshot 2024-02-07 131158](https://github.com/dani-garcia/vaultwarden/assets/400503/759ef18d-21eb-4db4-937c-61c22e53982f)

docker image: vaultwarden/server:latest

**Comment by BlackDex** [comment]
Try to fully logout and back in again.

**Reply by Vortilion** [reply]
Hey, already tried that. Actually it was the first thing I tried... This is my very first installation of the windows client.

**Reply by BlackDex** [reply]
So, can't you login, or sync??

I suggest to at least read https://bitwarden.com/help/change-client-environment/#tab-desktop-app-4dQ4hW1QAwVBuReXk2Txx0

Also, what errors do you see in the Vaultwarden logs?

If other clients are able to connect, it might be misconfiguration.

**Reply by Vortilion** [reply]
Hey, like I said in my post, I can login but I don’t see any items there, as if it were a completely new vault. Your link did not provide any new information. Setup on the mobile app was the same and there it works just fine. I will have to check the logs!

**Reply by BlackDex** [reply]
Also try to check the F12/Developer Console in the Desktop Client.
It might provide some more information.

**Reply by Vortilion** [reply]
It seems I'm getting a 404 in the Windows Client. Is it possible to edit the server settings anywhere? Maybe it was a typo somewhere... 
![Screenshot 2024-02-09 113419](https://github.com/dani-garcia/vaultwarden/assets/400503/54af826d-4a56-4d7a-b209-4e43d98bffa1)

**Comment by Vortilion** [comment]
Aaaah, I found the culprit (settings-wise, the real culprit is sitting in front of the monitor, as usual)... I added the server url in the custom settings as well for server api etc. ...

**Comment by kougyokugentou** [comment]
I'm getting the same here on Bitwarden desktop client 2024.2.0
404 to  https://server.domain.tld:PORT/accounts/prelogin
I tried pasting the server URL in all the custom settings with no luck.  I also tried desktop client 2024.1.0

Version 2024.2.0
Shell 28.1.3
Renderer 120.0.6099.199
Node 18.18.2
Architecture x64

**Reply by BlackDex** [reply]
You actually do not need to add it to all fields. That is what the OP's issue was in the first place.

**Reply by kougyokugentou** [reply]
Yes, I tried it in only the one field as well, still no luck no matter how I try to cut the cake.
![2024-02-13_10-08-40](https://github.com/dani-garcia/vaultwarden/assets/41278462/90b81e6e-ebf9-4db3-837d-ead4c9a43bd9)

**Reply by BlackDex** [reply]
What do the logs on the Vaultwarden side report during this action?

**Comment by kougyokugentou** [comment]
I believe I've found the issue and will need to figure out my internal CA business.
[2024-02-13 18:48:50.782][rocket_http::tls::listener][WARN] tls handshake with dockerBridgeIP:randomHighPort failed: received fatal alert: CertificateUnknown
